### PR TITLE
adding ngmix

### PR DESCRIPTION
**Pull request by beckermr**
This PR adds ngmix.

### CODE REVIEW

File: recipes/ngmix/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  home: https://github.com/esheldon/ngmix
+  license: GPL-2
+  license_family: GPL
+  license_file: gpl.txt
+  summary: 'Gaussian mixtures and image processing implemented in python and C'
+
+extra:
+  recipe-maintainers:
+    - beckermr
+    - esheldon
```
Comment (beckermr) [inline recipes/ngmix/meta.yaml]: @esheldon please confirm if you would like to help maintain this or not?

Comment (esheldon) [reply recipes/ngmix/meta.yaml]: I confirm

Comment (esheldon) [reply recipes/ngmix/meta.yaml]: I believe 0.42 also has the bug

Comment (beckermr) [reply recipes/ngmix/meta.yaml]: Ohhhh really. Mmk i'll bump to 0.43 then

File: recipes/ngmix/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,44 @@
+{% set name = "ngmix" %}
+{% set version = "1.2.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/esheldon/ngmix.git
```
Comment (CJ-Wright) [inline recipes/ngmix/meta.yaml]: There most likely should have been some discussion about this, we usually only accept tarball http://conda-forge.org/docs/maintainer/adding_pkgs.html?highlight=tarballs#tarballs-no-repos

**Pull request merged**